tests/coalaCITest.py

```diff
@@ -33,9 +33,7 @@ def test_nonexistent(self, debug=False):
         retval, stdout, stderr = execute_coala(
             coala.main, 'coala', '--non-interactive', '-c', 'nonex', 'test')
```

Comment:
you shouldnt make a commit just for tests, cuz if we ever revert a commit from the others you made, the tests will fail. you should make the test changes in the corresponding commits